website/content/docs/secrets/pki/considerations.mdx

```diff
@@ -445,7 +469,8 @@ For these personas, we suggest the following ACLs, in condensed, tabular form:
 | `/revoke` | Write | Yes | Yes | | | |
 | `/tidy` | Write | Yes | Yes | | | |
 | `/tidy-status` | Read | Yes | Yes | | | |
```

Comment:
Missing `/tidy-cancel` and I forgot to update this list with OCSP paths as well :(